setup.sh

```diff
@@ -1,56 +1,64 @@
 #!/bin/bash
-time=`date '+%Y%m%d_%H%M'`
-VAG_COMMON_DIR="/vagrant/"
+time=`TZ=Asia/Jerusalem date '+%Y%m%d_%H%M'`
+VAG_COMMON_DIR="/vagrant"
+HOME="/home/vagrant"
 LOG_DIR="$VAG_COMMON_DIR/log"
 OUTFILE="$LOG_DIR/${time}_vagrant-out.log"
+FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
+FLASK_DIR="$HOME/FlaskApp"
```

Comment:
What's the purpose of this directory? I see you are creating it further down, but I don't see it being used at all.

Reply:
The script changes directory to this dir prior to setting the virtual environment, and running Flask.
all related virtual environment files are located in this dir.

Reply:
Will be removed in next commit, as I removed python's virtual environments.